examples/next/faustwp-getting-started/faust.config.js

```diff
@@ -9,6 +9,6 @@ import { CustomToolbar } from './plugins/CustomToolbar.tsx';
 export default setConfig({
   templates,
   experimentalPlugins: [new CustomToolbar()],
+  enableExperimentalToolbar: false,
```

Comment:
Thinking that we may want to make this true in the example project so that it's available during development. Also shortened to `experimentalToolbar` for consistency with our experimental prefix

```suggestion
  experimentalToolbar: true,
```